projects/plugins/jetpack/modules/protect/blocked-login-page.php

```diff
@@ -264,82 +348,103 @@ function send_recovery_email() {
+			$content = '<style>html{ background-color: #fff; } #error-message { margin:0 auto; padding: 1em; box-shadow: none; } </style>' . $content;
 		}
 		$this->display_page( $title, $content, $back_link, $recovery_form );
 
 	}
 
-	function render_recovery_form() {
+	/**
+	 * Render the recovery form.
+	 */
+	public function render_recovery_form() {
 		$content = $this->get_html_blocked_login_message();
 		$this->protect_die( $content, null, null, true );
 	}
 
-	function render_recovery_success() {
+	/**
+	 * Render the recovery instructions.
+	 */
+	public function render_recovery_success() {
+		// translators: the email address the recovery email was sent to.
 		$this->protect_die( sprintf( __( 'Recovery instructions were sent to %s. Check your inbox!', 'jetpack' ), $this->email_address ) );
 	}
 
-
-	function get_html_blocked_login_message() {
+	/**
+	 * Get the HTML for the blocked login message.
+	 */
+	public function get_html_blocked_login_message() {
 		$icon = '<svg class="gridicon gridicons-spam" style="fill:#d94f4f" height="24" width="24" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><g><path d="M17 2H7L2 7v10l5 5h10l5-5V7l-5-5zm-4 15h-2v-2h2v2zm0-4h-2l-.5-6h3l-.5 6z"/></g></svg>';
-		$ip = str_replace( 'http://', '', esc_url( 'http://' . $this->ip_address ) );
+		$ip   = str_replace( 'http://', '', esc_url( 'http://' . $this->ip_address ) );
 		return sprintf(
-			__( '<p>Your IP address <code>%2$s</code> has been flagged for potential security violations. You can unlock your login by sending yourself a special link via email. <a href="%3$s">Learn More</a></p>', 'jetpack' ),
+			// translators: the IP address that was flagged.
+			__( '<p>Your IP address <code>%2$s</code> has been flagged for potential security violations. You can unlock your login by sending yourself a special link via email. <a href="%3$s">Learn More</a></p>', 'jetpack' ), // phpcs:ignore WordPress.WP.I18n.NoHtmlWrappedStrings
 			$icon,
 			$ip,
 			esc_url( self::get_help_url() )
 		);
 	}
 
-	function get_html_recovery_form() {
+	/**
+	 * Get the HTML recovery form.
+	 */
+	public function get_html_recovery_form() {
 		ob_start(); ?>
-        <div>
-            <form method="post" action="?jetpack-protect-recovery=true">
-				<?php echo wp_nonce_field( 'bypass-protect' ); ?>
-                <p><label for="email"><?php esc_html_e( 'Your email', 'jetpack' ); ?><br/></label>
-                    <input type="email" name="email" class="text-input"/>
-                    <input type="submit" class="button"
-                           value="<?php esc_attr_e( 'Send email', 'jetpack' ); ?>"/>
-                </p>
-            </form>
-        </div>
+		<div>
+			<form method="post" action="?jetpack-protect-recovery=true">
+				<?php echo wp_nonce_field( 'bypass-protect' ); //phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?> 
```

Comment:
Do we need to `echo` here? `wp_nonce_field` already echoes on its own.

```suggestion
				<?php wp_nonce_field( 'bypass-protect' ); ?> 
```